cmd/src/batch_exec.go

```diff
@@ -109,27 +108,11 @@ func executeBatchSpecInWorkspaces(ctx context.Context, ui *ui.JSONLines, opts ex
 	// we can convert it to a RepoWorkspace and build a task only for that one.
 	repoWorkspace := convertWorkspace(input.Workspace)
 
-	// Parse the raw batch spec contained in the input
-	ui.ParsingBatchSpec()
-	batchSpec, err := svc.ParseBatchSpec([]byte(input.RawSpec))
-	if err != nil {
-		var multiErr *multierror.Error
-		if errors.As(err, &multiErr) {
-			ui.ParsingBatchSpecFailure(multiErr)
-			return cmderrors.ExitCode(2, nil)
-		} else {
-			// This shouldn't happen; let's just punt and let the normal
-			// rendering occur.
-			return err
-		}
-	}
-	ui.ParsingBatchSpecSuccess()
-
```

Comment:
I love that this is now also gone. It was confusing to read that we're parsing this here (but we also parse it server-side, etc.)